telescope-wrapper

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+architecture="$(uname -p)"
+
+if [ $architecture = "x86_64" ] ; then
+  export QML2_IMPORT_PATH=`pwd`/usr/lib/x86_64-linux-gnu/qt5/qml #desktop
+else
+  export QML2_IMPORT_PATH=`pwd`/usr/lib/arm-linux-gnueabihf/qt5/qml/ #phone
+fi
+
+env
```

Comment:
I'm unsure why you run env and find. here, could you clarify that? And if it's not necessary, could you remove that?

Reply:
This is left from debugging and now removed. Thank you for reviewing my work!